ts/features/itwallet/common/store/reducers/preferences.ts

```diff
@@ -25,6 +26,8 @@ export type ItwPreferencesState = {
   isPendingReview?: boolean;
   // Indicates the SPID/CIE authentication level used to obtain the eid
   authLevel?: ItwAuthLevel;
+  // Indicates whether the claim values should be hidden for a specific credential type
+  claimValuesHiddenByCredential?: { [credentialType: string]: boolean };
```

Comment:
Why do we need to hide by credential type? Shouldn't be a feature credential-wise?
I think user expect to keep this preference store for every credential, even the ones that we may add in the future. What do you think?

```suggestion
  claimValuesHidden?: boolean;
```

Reply:
Addressed in [9f7da79](https://github.com/pagopa/io-app/pull/6742/commits/9f7da79d6024dbb0ebab2eab22bd65df9263bd31)  the updated implementation with claim visibility management generalized for all credentials, not just by credential type